src/xref2/scratch.md

```diff
@@ -125,7 +127,7 @@ let sg = Common.model_of_string_impl test_data;;
                     `Module
                       (`Result
                          (`Result (`Module (`Root (Common.root, "Root"), M))),
-                       E/hidden);
+                       $E);
```

Comment:
Is this change caused by the version of OCaml ? I'm using 4.08.1.

Reply:
No, this is due to the rebase - I've changed from using `/hidden` to using the `InternalName` things.